### PR TITLE
Added static registration landing page

### PR DESCRIPTION
**Pull request by TiagoGraf89**
Title says it all

### CODE REVIEW

**Comment by TiagoGraf89** [comment]
Link to SonarQube analysis
https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=7132-WebClient

File: Apps/WebClient/src/ClientApp/components/registration/registration.vue.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                    <h3>
+                        <strong>Use a Mobile Card</strong>
+                    </h3>
+                    <p>A mobile card is a representation of a BC Services Card on an iPhone, iPad, or Android mobile device.</p>
+                </div>
+            </div>
+            <div class="col-lg-6 col-md-6 pt-2">
+                <div class="shadow-box">
+                    <h3>
+                        <strong>Use a Card Reader & Passcode</strong>
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/components/registration/registration.vue.html]: These will need to be internationalized @bradhead

File: Apps/WebClient/src/ClientApp/components/registration/registration.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.shadow-box {
```
Comment (FuriousLlama) [inline Apps/WebClient/src/ClientApp/components/registration/registration.css]: Could this come from bootstrap instead?

Comment (TiagoGraf89) [reply Apps/WebClient/src/ClientApp/components/registration/registration.css]: Found something similar, updated

**Pull request merged**